### ISSUE TITLE
hc_init_globals function might not work, or be misleading for APP_AGENT_LATEST_HASH

### ISSUE THREAD

**Issue by Connoropolous** [holochain/holochain-rust#270]
This is because APP_AGENT_LATEST_HASH is not a constant but should change during use, after the agent calls `hc_update_agent`. Using it as a constant in hdk-rust might be flawed?

@ddd-mtl

**Comment by ddd-mtl** [comment]
Yeah, this might change when `hc_update_agent` will be implemented.

**Comment by Connoropolous** [comment]
I have an API change proposal for this issue then, in which alongside `hc_update_agent` we create a new function called `hc_agent_latest_hash` to retrieve their latest value. 

APP_AGENT_INITIAL_HASH could still be a constant since it inherently is one
